idioms-json-2.1/malicious-email-indicator-with-addresses.json

```diff
@@ -10,20 +35,36 @@
             ],
             "modified": "2014-10-31T15:52:13.126Z",
             "name": "Malicious E-mail Addresses",
-            "pattern": "[email-message:from_ref.value IN ('fred@abc.com', 'betty@abc.com') AND (email-message:to_refs[*].value = 'barney@abc.com' OR email-message:to_refs[*].value = 'wilma@abc.com')]",
+            "pattern": "[email-message:from_ref.value IN ('fred@abc.com', 'betty@abc.com') AND (email-message:to_refs[*].value = 'barney@abc.com' OR email-message:to_refs[*].value = 'wilma@abc.com') AND email-message:message_id = '20131031123417.u1BKfrXh004925@mail-gw-01.abc.com']",
```

Comment:
I see that the pattern does include the `email-message:message_id` but the actual `email-message` SCO has the property missing...

Happens only with the `malicious-email-indicator-with-addresses` examples.

Reply:
Its a little more complicated than that.  There are two almost similar observables - one in the indicator, the other outside (see the XML).  I just added the message_id to one of them - in the indicator.  So it appears in the pattern, bu not in the SCO.  If you think it is necessary I can make them more different...

Reply:
Ahh ok! Not necessary.